### PR TITLE
sys-fs/fragview: New package

### PR DESCRIPTION
**Pull request by lucianposton**
Closes: https://bugs.gentoo.org/662614
Package-Manager: Portage-2.3.40, Repoman-2.3.9

### CODE REVIEW

File: sys-fs/fragview/fragview-0.1.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+HOMEPAGE="https://github.com/i-rinat/fragview"
+
+LICENSE="MIT"
+SLOT="0"
+
+DEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:3.0
+	dev-db/sqlite:3
+	dev-libs/boost
```
Comment (hydrapolic) [inline sys-fs/fragview/fragview-0.1.1.ebuild]: Isn't it desirable to rebuild the tool on lib slot changes?

Comment (lucianposton) [reply sys-fs/fragview/fragview-0.1.1.ebuild]: yea, let me add that

File: sys-fs/fragview/files/fragview-0.1.1-fix-linking.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
+From 11ab3027fda015798b92fd7e44cb5e3289f41542 Mon Sep 17 00:00:00 2001
```
Comment (mgorny) [inline sys-fs/fragview/files/fragview-0.1.1-fix-linking.patch]: Please include link to upstream bug tracker so we can monitor its status.

File: sys-fs/fragview/fragview-0.1.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+DEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:3.0
+	dev-db/sqlite:3
+	dev-libs/boost:=
+"
+RDEPEND="${DEPEND}"
+
+PATCHES="${FILESDIR}/${P}-fix-linking.patch"
```
Comment (mgorny) [inline sys-fs/fragview/fragview-0.1.1.ebuild]: PATCHES should be an array.

Comment (lucianposton) [reply sys-fs/fragview/fragview-0.1.1.ebuild]: Why should it be an array? Briefly looking at the spec, it looks like it was written to support both, no?

Comment (mgorny) [reply sys-fs/fragview/fragview-0.1.1.ebuild]: Technically FILESDIR can contain spaces which breaks non-array form.

File: sys-fs/fragview/files/fragview-0.1.1-fix-linking.patch (hunk 2 above)
Comment (lucianposton) [reply sys-fs/fragview/files/fragview-0.1.1-fix-linking.patch]: done

File: sys-fs/fragview/fragview-0.1.1.ebuild (hunk 3 above)
Comment (lucianposton) [reply sys-fs/fragview/fragview-0.1.1.ebuild]: done

**Comment by mgorny** [comment]
Thank you, great work! I'm going to merge it in a minute.

**Pull request closed without merging**